zstacklib/zstacklib/utils/lvm.py

```
@@ -2774,6 +2775,142 @@ def get_lv_location(lv_path):
     return []
 
 
+def _run_lvm_json_report(command):
+    cmd = shell.ShellCmd(command)
+    cmd(is_exception=False)
+    if cmd.return_code != 0:
+        raise Exception("failed to query LVM range metadata, command[%s], error[%s]" %
+                        (command, cmd.stderr))
+    try:
+        return simplejson.loads(cmd.stdout)
+    except Exception:
```

Ruff (0.16.0)

[warning] Do not catch blind exception: `Exception`

(BLE001)